webgpu.h

```diff
@@ -767,9 +794,10 @@ typedef void (*WGPUProc)();
 #if !defined(WGPU_SKIP_PROCS)
 
 typedef WGPUInstance (*WGPUProcCreateInstance)(WGPUInstanceDescriptor const * descriptor);
-typedef WGPUProc (*WGPUProcGetProcAddress)(WGPUDevice device, const char* procName);
+typedef WGPUProc (*WGPUProcGetProcAddress)(WGPUDevice device, char const * procName);
 
 // Procs of Adapter
+typedef bool (*WGPUProcAdapterGetProperties)(WGPUAdapter adapter, WGPUAdapterProperties * properties);
```

Comment:
I chose to have `AdapterGetProperties` return a boolean so that it can return `false` when it doesn't support one of the extensions used in `properties`. We should figure out how we want error handling before `WGPUDevice` to happen.

Reply:
This isn't the way Vulkan does it, I think. Instead, the STYPE things not known are just ignored, and the spec is written in such a way that leaving their contents to be zeroed is equivalent to the defaults. I mean, this function isn't the right place to check for supported extensions anyway, we shouldn't try.

Reply:
I see, so you'd rather return `void` here?

Reply:
yeah

Reply:
#28